MapboxVision/VisionManager/VisionManager.swift

```diff
@@ -34,6 +42,8 @@ public final class VisionManager: BaseVisionManager {
     /**
      Start delivering events from `VisionManager`.
 
+     Do NOT call this method more than once or after `destroy` is called.
```

Comment:
nit: Should we wrap this into `- Important` section as we did below?

Reply:
Yep, thanks.